indico/core/celery/controllers.py

```diff
@@ -49,5 +50,6 @@ def _process(self):
                           'schedule': entry['schedule'],
                           'custom_schedule': custom_schedule,
                           'disabled': disabled})
+        tasks.sort(key=itemgetter('name'))
```

Comment:
I prefer to keep the alphabetical order over moving the disabled tasks at the bottom but since it is displayed that way when starting the worker, why not. I'll amend it quickly ;)